build/build-report.js

```diff
@@ -81,14 +85,18 @@ async function buildEsModulesBundle() {
     file: 'dist/report/bundle.js',
     format: 'esm',
   });
+  console.log('dist/report/bundle.js written');
 }
 
 if (require.main === module) {
-  if (process.argv[2] === '--only-standalone') {
+  if (process.argv.includes('--psi')) {
+    buildPsiReport();
+  } else if (process.argv.includes('--standalone')) {
```

Comment:
wdyt of breaking up these if/elses, and only doing the last block if there are no cli params?